src/main/java/teammates/ui/controller/AdminSessionsPageAction.java

```diff
@@ -30,141 +29,116 @@
-                logic.getAllOpenFeedbackSessions(rangeStartUtc, rangeEndUtc);
+        List<FeedbackSessionAttributes> allOpenFeedbackSessionsList = logic.getAllOpenFeedbackSessions(
+                TimeHelper.convertLocalDateTimeToInstant(rangeStart, timeZone),
+                TimeHelper.convertLocalDateTimeToInstant(rangeEnd, timeZone)
+        );
 
         result = createShowPageResultIfNoOngoingSession(allOpenFeedbackSessionsList);
         if (result != null) {
             return result;
         }
 
-        result = createAdminSessionPageResult(allOpenFeedbackSessionsList);
-
-        return result;
-
+        return createAdminSessionPageResult(allOpenFeedbackSessionsList);
     }
 
-    private void putIntoUnknownList(
-            Map<String, List<FeedbackSessionAttributes>> map, FeedbackSessionAttributes fs) {
-        if (map.get("Unknown") == null) {
-            List<FeedbackSessionAttributes> newList = new ArrayList<>();
-            newList.add(fs);
-            map.put("Unknown", newList);
-        } else {
+    private void putIntoUnknownList(Map<String, List<FeedbackSessionAttributes>> map, FeedbackSessionAttributes fs) {
+        if (map.get("Unknown") != null) {
             map.get("Unknown").add(fs);
+            return;
         }
+
+        List<FeedbackSessionAttributes> newList = new ArrayList<>();
+        newList.add(fs);
+        map.put("Unknown", newList);
     }
 
-    private void prepareDefaultPageData(Calendar calStart, Calendar calEnd) {
+    private void prepareDefaultPageData(LocalDateTime start, LocalDateTime end) {
         this.map = new HashMap<>();
         this.totalOngoingSessions = 0;
         this.totalOpenStatusSessions = 0;
         this.totalClosedStatusSessions = 0;
         this.totalOpenStatusSessions = 0;
         this.totalInstitutes = 0;
-        this.rangeStart = calStart.getTime();
-        this.rangeEnd = calEnd.getTime();
+        this.rangeStart = start;
+        this.rangeEnd = end;
     }
 
-    private ActionResult createShowPageResultIfParametersInvalid() {
-        String startDate = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_STARTDATE);
-        String endDate = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_ENDDATE);
-        String startHour = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_STARTHOUR);
-        String endHour = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_ENDHOUR);
-        String startMin = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_STARTMINUTE);
-        String endMin = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_ENDMINUTE);
-        String timeZone = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_TIMEZONE);
+    private ActionResult validateParametersAndCreateShowPageResultIfInvalid() {
+        ZoneId zone = Const.DEFAULT_TIME_ZONE;
+        LocalDateTime start = LocalDateTime.now(zone).minusDays(3);
+        LocalDateTime end = LocalDateTime.now(zone).plusDays(4);
```

Comment:
I think these 3 variables are redundant and unnecessarily gives rise to lines 118-120 as well as the parameters of `prepareDefaultPageData`. I think we should just work on `this.rangeStart` etc. directly